src/components/sbb-header/readme.md

```diff
@@ -0,0 +1,110 @@
+# sbb-header
+
+The `sbb-header` component is a container for actions and logo, and it is displayed sticky at page's top.
+
+It has two slots: 
+the first one can contain one or more [sbb-header-action](../sbb-header-action/readme.md) or other action items
+like `sbb-button` or `sbb-link`, and it is displayed at the left end of the component; the second slot is displayed
+at the right end, and it can contain a logo, which by default is the [sbb-logo](../sbb-logo/readme.md).
+
+A box-shadow appears under the component if the `shadow` variable is set to `true`. 
+
+The component's height can be overridden by defining the variable `--sbb-header-height-override`.
+
+## Style
+
+Users can customize position and behaviour of actions inside the `sbb-header` component 
+by adding classes to `sbb-header-action` elements and then defining their own style rules.
+
+An example has been created with the following requirements:
+- 4 action item (with custom icons);
+- the first item is always left aligned and has `expand-from` set to `small`;
+- the other 3 items are left aligned in breakpoints zero to medium, and right aligned from large to ultra;
+- the last item is not visible in breakpoints zero to small.
+
+To achieve this result, a `div` tag with a class named `sbb-header__spacer` was added between the first 
+and the second `sbb-header-action` item, then a class named `sbb-header-action__last-element` was added to the last one.
+Finally, the following custom CSS has been added*. The result can be seen in the home and home--logged-in stories.
+
+```css
+.sbb-header__spacer {
```

Comment:
Nitpick; Our consumers do not need to consider BEM. Should we maybe simplify the documentation to just `.spacer` and `.last-action`?